src/salesforce_uploader.js

```diff
@@ -35,29 +37,44 @@ type apiGatewayLambdaInput = {
   headers: inputHeaders
 
 }
+
+async function copyToUploadedFolder (stage, s3Path, sfFileName) {
+  try {
+    let uploadedPath = `${stage}/uploaded/${sfFileName}`
+    await
+      copyObject(s3Path, uploadedPath)
+  } catch (err) {
+    console.error('error copying fulfilment file to uploaded directory')
+    console.log(err)
```

Comment:
do you mean safe as in this will not throw another exception when trying to log to console or safe as in we are not logging private information?
I think it should be safe, but the reason I'm just logging directly is that I don't want to make any assumptions about the type of the err variable